### ISSUE TITLE
How to change fieldOfView

### ISSUE THREAD

**Issue by raminia** [allenai/ai2thor#34]
I realized the field Of View is fixed and the interface does not support changing it. Is there any specific reason for this limitation or this is just not exposed?

**Comment by ekolve** [comment]
You might be able to get what you need by adjusting the width of the player.  Can you try starting the environment as follows:
```python
controller.start(player_screen_width=2000) # this can be any number >=300
```
That should give you a pretty large field of view

**Comment by raminia** [comment]
How about vertical FOV?

**Comment by ekolve** [comment]
```python
controller.start(player_screen_height=2000) # 
```

**Comment by ekolve** [comment]
though if you want a square aspect ratio and a larger field of view, we would have to parameterize some way for the camera position to be modified - its possible just not exposed.

**Comment by ElizabethDuo** [comment]
One more question about FOV. Now in thor it is first-person view, can I change it into third-person view?

**Comment by ekolve** [comment]
@ElizabethDuo - could you open a separate issue to discuss a third person camera view?

**Comment by ElizabethDuo** [comment]
@ekolve already open issue #65, thanks for your attention!

**Comment by ekolve** [comment]
In version 0.044 (just released), I have added fieldOfView as a parameter to the Initialize action.  Please take a look here http://ai2thor.allenai.org/tutorials/actions.  The value corresponds to this value on the Camera in Unity3d: https://docs.unity3d.com/ScriptReference/Camera-fieldOfView.html

**Comment by ekolve** [comment]
Fixed with version 0.0.44
